src/components/tooltip/TooltipDirective.js

```diff
@@ -44,7 +44,8 @@ goog.require('ga_topic_service');
             '<div ng-bind-html="html.snippet"></div>' +
             '<div ng-if="::html.showVectorInfos" class="ga-vector-tools">' +
               '<div ga-measure="::html.feature"></div>' +
-              '<div ga-profile-bt="::html.feature"></div>' +
+              '<div ng-if="::!html.showProfile" ' +
```

Comment:
I am referring to https://github.com/geoadmin/mf-geoadmin3/issues/3471#issuecomment-244926101, but I am fine with leaving it. it's not really optimized and our profile is fragile